ideascube/mediacenter/templates/mediacenter/document_detail.html

```diff
@@ -19,10 +19,19 @@
 {% endblock twothird %}
 {% block third %}
     {% if user.is_staff %}
-        <ul class="card tinted admin">
-            <li>{% fa 'pencil' 'fa-fw' %}<a href="{% url 'mediacenter:document_update' pk=document.pk %}">{% trans 'Edit' %}</a></li>
-            <li>{% fa 'trash' 'fa-fw' %}<a href="{% url 'mediacenter:document_delete' pk=document.pk %}">{% trans 'Delete' %}</a></li>
-        </ul>
+        {% if document.package_id %}
+            <span class="card admin">
+               {% blocktrans %}
+                   Cannot edit or delete this document.
+                   It is installed by the {{ document.package_id }} package.
+               {% endblocktrans %}
```

Comment:
So I have to wonder... We're slowly going to a future where all content in Ideascube is managed by packages, including content created and uploaded by users.

Which pretty much means that the above block will be always visible, and no content will be modifiable.

That doesn't feel right, wouldn't it be better to let users modify the content, improve it, adapt it and share it freely?

Or is this just temporary, and eventually the content will be modifiable again, with modifications simply becoming package updates?